src/shared/utils/transformSessionsToCinemasWithTimeslots.ts

```diff
@@ -0,0 +1,25 @@
+import { Session } from "entities/session/model/types";
```

Comment:
when you import something from top layer to the bottom layer - is a sign for you to not put the logic here Eg here you created shared util method which imports something from top layer entities, so it means that this method is not fully shared anymore, because has relation with Session entity 
Think where you can move this util method